app/views/external_users/claims/_date_attended_fields_condenced.html.haml

```diff
@@ -0,0 +1,23 @@
+%fieldset
+  %legend.visuallyhidden
+  .grid-row.extra-data.nested-fields.fee-dates
+
+    - if defined?(parent_model_prefix) && defined?(submodel_count)
+      - error_prefix = "#{parent_model_prefix}_date_attended_#{submodel_count}"
+    - else
+      - error_prefix = "date_attended_0"
+
+    .column-one-third.fee-date-from{class: error_class?(@error_presenter, "#{error_prefix}_date")}
+      %span.auto-width{id: "#{error_prefix}_date"}
+        = f.label :date, t('.date_from')
+        = f.gov_uk_date_field :date
+        = validation_error_message(@error_presenter, "#{error_prefix}_date")
+
+    .column-one-third.fee-date-to{class: error_class?(@error_presenter, "#{error_prefix}_date_to")}
+      %span.auto-width{id: "#{error_prefix}_date_to"}
+        = f.label :date, t('.date_to')
+        = f.gov_uk_date_field :date_to
+        = validation_error_message(@error_presenter, "#{error_prefix}_date_to")
+
+    .column-one-third.fee-date-remove
+      = link_to_remove_association t('.remove'), f
```

Comment:
Related with this [comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2362/files#r187652882), seems like there's already some locales define for it?